asserts/pool.go

```diff
@@ -273,6 +368,22 @@ func (p *Pool) phase(ph poolPhase) error {
 	return nil
 }
 
+func (p *Pool) AddSequenceToUpdate(unresolved *AtSequence, group string) error {
```

Comment:
this needs doc to describing exactly how it behaved depending if AtSequence is pinned or not I suppose?

Reply:
I notice there's no AddUnresolvedSequence ? is the plan always to use AddSequenceToUpdate?  that internally would work but what current comment might not be complete... otoh the asymmetry is a bit confusing I think for users fo rthis

Reply:
Yes, that was my idea because I noticed they ended up practically identical. But it's possible I got something else wrong per your other comments, need to think.

Reply:
Also, if (a)symmetry is a concern, then having two functions makes sense.